src/main/scala/org/broadinstitute/dsde/workbench/sam/dataAccess/LdapRegistrationDAO.scala

```diff
@@ -89,13 +92,13 @@ class LdapRegistrationDAO(
                 new Attribute(Attr.member, subjectDn(subject)))), "enableIdentity-add", samRequestContext).void
         case ldape: LDAPException if ldape.getResultCode == ResultCode.ATTRIBUTE_OR_VALUE_EXISTS => IO.unit
       }
-    }
+    }.withInfoLogMessage(s"Enabled $subject in LDAP")
 
   override def disableIdentity(subject: WorkbenchSubject, samRequestContext: SamRequestContext): IO[Unit] = {
     executeLdap(IO(ldapConnectionPool.modify(directoryConfig.enabledUsersGroupDn, new Modification(ModificationType.DELETE, Attr.member, subjectDn(subject)))).void, "disableIdentity", samRequestContext).recover {
       case ldape: LDAPException if ldape.getResultCode == ResultCode.NO_SUCH_ATTRIBUTE
         || ldape.getResultCode == ResultCode.NO_SUCH_OBJECT =>
-    }
+    }.withInfoLogMessage(s"Disabled $subject in LDAP")
```

Comment:
Do these WorkbenchSubject objects risk leaking any pii into logs?

Reply:
Subject IDs are not PII